src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -916,10 +917,35 @@ private void OutputListToOutputWindow(string prefix, IEnumerable<string> values)
                     continue;
                 }
 
-                if (addedReferencePaths.Add(path))
+                if (metadataReferencesToAdd.TryGetValue(path, out var existingProperties))
                 {
-                    projectContext.AddMetadataReference(path, reference.Properties);
+                    // Merge existing aliases the properties that are already there
```

Comment:
typo? Seems like you meant, "Merge existing aliases **with** the properties that are already there"